_content/booklets/dhamma-in-english_suchart

```diff
@@ -0,0 +1,18 @@
+---
+title: "Dhamma in English 2014 & 2015"
+authors:
+  - suchart
+external_url: "https://www.mediafire.com/file/vmxkl1yx2mii4lu/DHAMMA_IN_ENGLISH_2016_.pdf/file"
```

Comment:
Wrong year 😊
This is the correct link: https://www.mediafire.com/file/by2zn5n34c48nmw/DHAMMA_IN_ENGLISH_2014%262015.pdf/file